src/components/Menu-Badge/MenuBadge.tsx

```diff
@@ -0,0 +1,100 @@
+import { Box, Typography } from '@mui/material';
+
+import { COLOR } from './MenuBadge.constraints';
+
+type MenuBadgeProps = {
+  label: string;
+  variant?: 'default' | 'color';
+};
+
+export const MenuBadge: React.FC<MenuBadgeProps> = ({ label, variant = 'default' }) => {
+  const isColor = variant === 'color';
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        alignItems: 'center',
+        gap: '1.8rem',
```

Comment:
Hardcoded gap `'1.8rem'` in sx. Project rules require using theme spacing tokens and avoid hardcoded px/rem values in styles.

**Sugerencia:**
```typescript
Use theme spacing via SxProps or numeric spacing, e.g., `gap: 2` or `gap: theme.spacing(2)` depending on your conventions.
```